packages/quantic/force-app/main/default/lwc/quanticResultAction/quanticResultAction.js

```diff
@@ -65,11 +53,35 @@ export default class QuanticResultAction extends LightningElement {
   _selected;
   /** @type {boolean} */
   _loading;
+  
+  /**
+   * Specifies whether the result action button is in selected state or not.
+   * @api
+   * @type {boolean}
+   */
+  @api 
+  get selected() {
+    return this._selected;
+  }
 
-  connectedCallback() {
-    this.setSelected(this.selected);
-    this.setLoading(this.loading);
+  set selected(isSelected) {
+    this._selected = isSelected;
+  }
 
+  /**
```

Comment:
We'll need to update the unit tests of this component to reflect these changes, I can help with this!

Reply:
And the E2E tests of the QuanticResultCopyToClipboard component x)